### PR TITLE
docs: add tony bove to about page

### PR DESCRIPTION
**Pull request by rockument69**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Tony Bove, a new member, is not listed in the Angular Contributors page.
Issue Number: N/A

## What is the new behavior?
Tony Bove photo and bio now included in the Angular Contributors page.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x ] No

### CODE REVIEW

**Review by aikithoughts — APPROVED** [review]

**Review by aikithoughts — APPROVED** [review]

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -667,6 +667,13 @@
     "groups": ["Angular"],
     "lead": "dennispbrown"
   },
+  "rockument69": {
+    "name": "Tony Bove",
+    "picture": "tonybove.jpg",
```
Comment (IgorMinar) [inline aio/content/marketing/contributors.json:672]: can you please rename the file to use your username? thank you!

```suggestion
    "picture": "rockument69.jpg",
```

File: aio/content/marketing/contributors.json [hunk 2]
```diff
@@ -667,6 +667,13 @@
...
+    "bio": "Tony is a technical writer with Expert Support. His lifelong passions are helping people use technology, writing fiction, and playing music. When he's not working or playing the harmonica with friends in a bluegrass band, he's swimming and snorkeling on a Kauai beach and playing ball with his Irish Wolfhound. He's worked at home for decades before it became a thing.",
+    "groups": ["Angular"],
+    "lead": "dennispbrown"
```
Comment (IgorMinar) [inline aio/content/marketing/contributors.json:675]: My understanding that your technical lead is @aikidave (lead here is meant "TL" and not the "manager")
```suggestion
    "lead": "aikidave"
```

**Review by IgorMinar — CHANGES_REQUESTED** [review]
Hi Tony!

Welcome to Angular!

Can you please do me a big favor a fix some of the issues I pointed out below.

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (rockument69) [reply aio/content/marketing/contributors.json]: OK, included in the following commit.

File: aio/content/marketing/contributors.json (hunk 2 above)
Comment (rockument69) [reply aio/content/marketing/contributors.json]: OK, included in the following commit.

Comment (rockument69) [reply aio/content/marketing/contributors.json]: Thanks for the welcome! Happy to be on board.

**Review by IgorMinar — APPROVED** [review]
Excellent. Thanks for the updates!

**Pull request closed without merging**